tests/test_api_base.py

```diff
@@ -36,7 +36,7 @@
 import subprocess
 from six.moves import zip_longest
 from tests import helpers
-from bmaptools import BmapHelpers, BmapCreate, Filemap
+from bmaptool import BmapHelpers, BmapCreate, Filemap
```

Comment:
Above there were some commits with `from . import`. I prefer `from bmaptool import` but I would also be happy to have an opinionated formatter like `ruff` or `black` decide for us.

Reply:
black (my preference for formatter) doesn't prevent `.` in an import. In general, guidance is usually use relative imports in the actual code, and named imports in the test cases (which, if you structure the repo correctly, don't have to be pushed to pypi).

Reply:
I honestly would rather have the formatter (`black` is fine) just do it for us. Done. :)

Reply:
I think what @JPEWdev is trying to say is that the formatter isn't going to give an opinion in this case. And that does appear to be what happens. I ran `black` over `bmaptool/BmapCreate.py` every combination of `from .BmapHelpers`, `from bmaptool.BmapHelpers`, and `from . import` and `from bmaptool import` and in none of those cases did `black` complain; even when I mixed and matched between names and dots. So `black` doesn't care to correct any usage of these lines.

Given the choice I think using the name looks nicer and is explicit. But if the general guidance is to use relative references then I'm happy to go that way too. Problem is my research indicates there is no right answer.

Reply:
There isn't. What you have is fine